src/app/applications/multicast/multicast-edit/multicast-edit.component.ts

```diff
@@ -181,8 +201,7 @@ export class MulticastEditComponent implements OnInit {
     this.scrollToTopService.scrollToTop();
   }
   ngOnDestroy(): void {
```

Comment:
tslint warns of OnDestroy not implemented even though you're using one of its lifecycle hooks